contracts/mocks/ERC7739SignerRSA.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {RSA} from "../utils/cryptography/RSA.sol";
+import {ERC7739Signer} from "../utils/cryptography/draft-ERC7739Signer.sol";
+import {EIP712} from "../utils/cryptography/EIP712.sol";
+
+contract ERC7739SignerRSA is ERC7739Signer {
+    bytes private _e;
+    bytes private _n;
```

Comment:
Are these storage accesses 4337 compliant ?

Reply:
This contract is not meant for ERC4337 but for general ERC7739 message signing. When used as part of an ERC4337 Account, storage accesses are allowed to the account storage itself.

The problem is when an account uses another contract's `isValidSignature` during the validation phase. In that case the transaction can still be processed privately (i.e. through any provider), is just that they won't rely it to the canonical mempool

Reply:
Its a mock anyway, lets resolve that.

It will come back if we ever build a 7579 module that relies on 1271 signature that are resolved by 7739 Signers